lib/carrierwave/storage/fog.rb

```diff
@@ -0,0 +1,13 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider               => 'AWS',                        # required
+    :aws_access_key_id      => 'xxx',                        # required
+    :aws_secret_access_key  => 'yyy',                        # required
+    :region                 => 'eu-west-1',                  # optional, defaults to 'us-east-1'
+    :host                   => 's3.example.com',             # optional, defaults to nil
+    :endpoint               => 'https://s3.example.com:8080' # optional, defaults to nil
```

Comment:
Line is too long. [88/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.